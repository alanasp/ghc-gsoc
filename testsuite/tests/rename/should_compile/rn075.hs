{-# OPTIONS_GHC -fwarn-warnings-deprecations #-}

-- Test deprecation of variable and data constructor exports

module ShouldCompile where

import Rn075_A
import Rn075_B
import Prelude hiding ( head )  -- to force head to be deprecated

f = head

