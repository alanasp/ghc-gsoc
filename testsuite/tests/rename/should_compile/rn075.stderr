rn075.hs:11:5: warning: [-Wdeprecations (in -Wdefault)]
    In the use of ‘head’
    (imported from Rn075_B, but defined in GHC.List):
    Deprecated: "B: head is deprecated "
